devito/dle/blocking_utils.py

```diff
@@ -38,7 +39,7 @@ def fold_blockable_tree(node, exclude_innermost=False):
             if any(not is_foldable(j) for j in pairwise_folds):
                 continue
             # Maybe heuristically exclude innermost Iteration
-            if exclude_innermost is True:
+            if blockinner is False:
```

Comment:
Does that describe a block that is inner to s-th else or does it describe the inner of a block? Or s-th else?

Reply:
Ok, it is for loop-blocking, refers to verb. It is fine.